packages/react/src/context/slash-id-context.tsx

```diff
@@ -123,7 +123,17 @@ export const SlashIDProvider: React.FC<SlashIDProviderProps> = ({
 
     user.logout();
     setUser(undefined);
-  }, [user, state]);
+    // we need to set the oid back to the root on log out
+    setOid(initialOid);
+    // we can't reinitialize the SDK here, as it picks up the old token and creates a new user immediately
+    sidRef.current = new SlashID({
```

Comment:
@kaspermroz would it make sense to wrap this behind a factory function like `createSlashID(options)`? Looks like it happens in two places and there is some logic behind the options and default values.